dotCMS/src/main/docker/original/Dockerfile

```diff
@@ -60,13 +60,12 @@ RUN apt update && \
 
 # Install PostgreSQL client and pg_dump
 RUN apt update && \
-    apt install -y --no-install-recommends wget gnupg2 lsb-release && \
-    echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
-    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
+    apt install -y --no-install-recommends postgresql-common && \
+    /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh -y && \
     apt update && \
     apt install -y --no-install-recommends postgresql-client-16 && \
     /usr/bin/pg_dump --version || exit 1 && \
-    apt purge -y wget gnupg2 lsb-release && \
+    apt purge -y postgresql-common && \
```

Comment:
do we need postgres common?  does this remove pg_dump?

The test for any new image should be to go into the backend and download a database dump of the instance.